src/components/header.js

```diff
@@ -1,19 +1,26 @@
 import { NavLink } from 'react-router-dom';
+import avatar from '../avatar.svg';
 
 const Header = () => (
-  <header>
-    <h2>Bookstore CMS</h2>
-    <nav>
-      <ul>
-        <li>
-          <NavLink to="/">BOOKS</NavLink>
-        </li>
-        <li>
-          <NavLink to="/categories">CATEGORIES</NavLink>
-        </li>
-      </ul>
-    </nav>
-  </header>
+  <>
+    <header>
+      <nav className="navbar">
+        <h2 className="logo">Bookstore CMS</h2>
+        <ul>
+          <li className="books-link">
+            <NavLink to="/">BOOKS</NavLink>
+          </li>
+          <li className="categories">
+            <NavLink to="/categories">CATEGORIES</NavLink>
+          </li>
+        </ul>
```

Comment:
Your project is almost complete. Good job so far. However, kindly make sure that your design matches the actual one in terms of color. The color of your design looks slightly different to the actual design as shown below. Kindly ensure you use the specified color of #121212

### Your design 

![Screenshot from 2023-09-29 19-38-17](https://github.com/Uzair-Manzoor/bookstore/assets/65233860/0396178c-78e8-448e-b662-7524d8ddf7ae)

### Actual design 

![Screenshot from 2023-09-29 19-34-37](https://github.com/Uzair-Manzoor/bookstore/assets/65233860/cf7a0c85-80fb-4367-a231-fc5e54be69d9)